package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "main": "node_modules/expo/AppEntry.js",
   "scripts": {
-    "start": "expo start",
+    "start": "concurrently \"node index.js\" \"expo start --ios\"",
```

Comment:
why was this change needed? now the default script is running for ios setting and it requires XCode to be installed

Reply:
start will now link to web, I used this command because I needed some way to start the backend temporarily and then run the frontend